windows/src/desktop/kmbrowserhost/kmbrowserhost.dproj

```diff
@@ -59,6 +59,9 @@
         <UWP_DelphiLogo44>$(BDS)\bin\Artwork\Windows\UWP\delphi_UwpDefault_44.png</UWP_DelphiLogo44>
         <UWP_DelphiLogo150>$(BDS)\bin\Artwork\Windows\UWP\delphi_UwpDefault_150.png</UWP_DelphiLogo150>
         <SanitizedProjectName>kmbrowserhost</SanitizedProjectName>
+        <DCC_Define>SENTRY_NOVCL;$(DCC_Define)</DCC_Define>
+        <VerInfo_Locale>3081</VerInfo_Locale>
+        <VerInfo_Keys>CompanyName=;FileDescription=$(MSBuildProjectName);FileVersion=1.0.0.0;InternalName=;LegalCopyright=;LegalTrademarks=;OriginalFilename=;ProgramID=com.embarcadero.$(MSBuildProjectName);ProductName=$(MSBuildProjectName);ProductVersion=1.0.0.0;Comments=</VerInfo_Keys>
```

Comment:
Is it OK for `ProductVersion` to remain 1.0.0.0?

Reply:
Yes, this is ignored in the build.